source/clear-linux/get-started/bootable-usb/bootable-usb.rst

```diff
@@ -185,12 +186,22 @@ Burn the |CL| image onto a USB drive
 
 #. Click the :guilabel:`START` button. See Figure 2.
 
-   .. figure:: figures/bootable-usb-windows-3.png
+   .. figure:: figures/bootable-usb-windows-02.png
       :scale: 80 %
       :alt: Rufus utility
 
       Figure 2: Rufus utility
 
+#. When the dialogue appears, select
+   :guilabel:`Write in ISO image mode (Recommended)`. See Figure 3.
+
+   .. figure:: figures/bootable-usb-windows-03.png
```

Comment:
Non blocking for this PR, but would like to share my thinking.

On Rufus, the .IMG defaulted to GPT partition scheme, UEFI only. 

With the switch to ISO, Rufus defaults to MBR partition scheme allowing for BIOS and UEFI booting. I'm thinking it would cause less errors/confusion if we explicitly say to select GPT in the documentation. That way, if someone tries to boot Clear on a legacy BIOS system it will fail faster (and often with a more obvious error message depending on the EFI).

Reply:
@puneetse, Let me check the default menu options in Rufus when I return to the office.

Reply:
The ISO actually supports Legacy BIOS firmware.
Using the default settings (IIRC, MBR), I have tested rufus on both legacy and uefi with success.